src/pages/admin/create-notes.tsx

```diff
@@ -45,6 +45,7 @@ interface State {
   notesList: any
   selectedTags: any
   noteObject: any
+  noteEditObject: any
```

Comment:
I strongly recommend writing real types instead of using `any`. It will greatly help new developers to the code to better understand what the data models are. I also recommend always using the nullable syntax `foo?: T` to show which variables can be null.